L1Trigger/L1THGCal/interface/concentrator/HGCalConcentratorProcessorSelection.h

```diff
@@ -13,7 +13,7 @@
 
 class HGCalConcentratorProcessorSelection : public HGCalConcentratorProcessorBase {
 private:
-  enum SelectionType { thresholdSelect, bestChoiceSelect, superTriggerCellSelect };
+  enum SelectionType { thresholdSelect, bestChoiceSelect, superTriggerCellSelect, mixedSelect };
```

Comment:
`mixedSelect` might not be explicit enough. It would be clearer to have something like `mixedBestChoiceSuperTriggerCell`